public/app.css

```diff
@@ -0,0 +1,15 @@
+html {
+  height: 100%;
+}
+body {
+  height: 100%;
+  margin: 0;
+  font-family: fantasy;
```

Comment:
Can we add some fallback fonts just in case "fantasy" isn't supported in all browsers? i.e. helvetica, arial, sans-serif...